out_mongo.go

```diff
@@ -72,7 +61,7 @@ func FLBPluginFlush(data unsafe.Pointer, length C.int, tag *C.char) int {
 		}
 
 		collectionName := strings.Replace(fmt.Sprintf("%s_%s_%s", logDoc.Customer, logDoc.PlatformId, logDoc.ProjectId), "-", "_", -1)
-		collection := session.DB(mongoConfig.Database).C(collectionName)
```

Comment:
Define constant (e.g. MONGO_DB_DEFAULT) with empty string and use it. As default DB is set into `DialInfo`